spec/nandi/config_spec.rb

```diff
@@ -0,0 +1,136 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+require "nandi/config"
+
+RSpec.describe Nandi::Config do
+  subject(:config) { described_class.new }
+
+  before do
+    Nandi.instance_variable_set(:@config, nil) # Reset config
+    config.lockfile_directory = "db"
+  end
+
+  context "multi-database integration" do
+    before do
+      config.register_database(
+        :primary,
+        migration_directory: primary_migration_directory,
+        output_directory: primary_output_directory,
+      )
+      config.register_database(
+        :analytics,
+        migration_directory: analytics_migration_directory,
+        output_directory: analytics_output_directory,
+      )
+    end
+
+    let(:primary_migration_directory) { "db/primary_migrations" }
```

Comment:
```suggestion
    let(:primary_migration_directory) { "db/migrations" }
```
should it be this? Same with the output dir below

Reply:
Just checking that setting the values explicitly is working here, will rename so that we're definitely not covering the prefix case 👍